src/translator.h

```diff
@@ -1,4 +1,4 @@
-﻿/******************************************************************************
+/******************************************************************************
  *
  *
```

Comment:
Strange, I checked the code base I have and I also created a small project (and ran it with the MSCVC 19 64-bit compiler).
I found no indication that a file was written into UTF-8 BOM mode as all HTML files gave (with the Cygwin `file` command):
```
HTML document, Unicode text, UTF-8 text
```
I converted one file (with Notepad++) to UTF-8 BOM mode and this converted fiel gave:
```
HTML document, Unicode text, UTF-8 (with BOM) text
```
must be something that slipped in (with one or your edits?).
Are the other files you changes have similar problems?

Reply:
I building doxygen from windows command line by this:
```
mkdir build
cd build
cmake -G "NMake Makefiles" ..
nmake
```

So you can repeat same situating on your side. Simply add BOM to the file: `translator_cz.h` build all with `nmake`.
Set in the `Doxyfile` option `OUTPUT_LANGUAGE        = Czech`.
And build documentation by doxygen. Result html pages contains in menu `?` characters.

![doxygen_utf8_bom_localization_problem](https://user-images.githubusercontent.com/89853747/187729211-ceb2fde4-edcf-4bac-a705-3a11dd652f57.png)

Reply:
With the version I have I changed the `translator_cz.h` into a UTF-8 BOM mode version, compiled and I got also the wrong characters. Looking at the UTF-8 BOM mode version of `translator_cz.h` I see here (with the Cygwin od -bc`):
```
0000000 357 273 277 057 052 052 052 052 052 052 052 052 052 052 052 052
        357 273 277   /   *   *   *   *   *   *   *   *   *   *   *   *
```
in the original version these lines are:
```
0000000 057 052 052 052 052 052 052 052 052 052 052 052 052 052 052 052
          /   *   *   *   *   *   *   *   *   *   *   *   *   *   *   *
```
so the UTF-8 BOM mode version contains some extra characters and indication.

Probably the problem slipped in in your version, but anyway good that you found the problem.

Reply:
Did some thinking based on this nasty problem. Maybe we have to introduce in the one of the GitHub (*nix) Actions  a test on the files from the repository that would signal such a problem and have the process fail.